engine/src/main/java/io/zeebe/engine/state/deployment/DbProcessState.java

```diff
@@ -99,11 +102,26 @@ public DbProcessState(
 
   @Override
   public void putDeployment(final DeploymentRecord deploymentRecord) {
-    for (final ProcessRecord processRecord : deploymentRecord.processes()) {
-      putProcess(processRecord.getKey(), processRecord);
+    for (final DeployedProcessMetadata metadata : deploymentRecord.processes()) {
+      for (final DeploymentResource resource : deploymentRecord.getResources()) {
+        if (resource.getResourceName().equals(metadata.getResourceName())) {
+          processRecordForDeployments.reset();
+          processRecordForDeployments.wrap(metadata, resource.getResource());
+          putProcess(metadata.getKey(), processRecordForDeployments);
+        }
+      }
     }
   }
 
+  @Override
+  public void putLatestVersionDigest(
+      final DirectBuffer processIdBuffer, final DirectBuffer digest) {
+    processId.wrapBuffer(processIdBuffer);
+    this.digest.set(digest);
+
+    digestByIdColumnFamily.put(processId, this.digest);
+  }
+
```

Comment:
Sorry for these, it seems this file doesn't align with our defined order. The only change I made was in the putDeployment.

Reply:
What I've done is separate these changes into a different commit so the reviewer can "ignore" them. But this is pretty small anyway, all good :+1:

Reply:
Yeah true last time I actually did sorry.